python/looker_sdk/rtl/requests_transport.py

```diff
@@ -64,31 +65,29 @@ def request(
         headers: Optional[MutableMapping[str, str]] = None,
         transport_options: Optional[transport.TransportSettings] = None,
     ) -> transport.Response:
-
-        url = f"{self.api_path}{path}"
```

Comment:
I think maybe this whole change could boil down to the following (provided we're ok with continuing to let `requests` handle the `query_params` rather than doing it ourselves)

```python
url = path if urllib.parse.urlparse(path).scheme else f"{self.api_path}{path}"
```

Reply:
This is great, nice catch!